engine/src/main/java/org/terasology/world/chunks/Chunks.java

```diff
@@ -325,8 +325,8 @@ public static Vector3i toRelative(Vector3ic worldPos, Vector3i dest) {
      * @param worldPos world position
      * @return the modified {@code worldPos} vector
      */
-    public static Vector3i toRelative(Vector3i worldPos) {
-        return toRelative(worldPos, worldPos);
+    public static Vector3i toRelative(Vector3ic worldPos) {
+        return toRelative(worldPos, new Vector3i(worldPos));
```

Comment:
We've been taking the decision to no longer hide those defensive copies of vectors we were doing all over the place. This, along with other methods, deliberately takes a mutable vector because it will modify it in-place. 

Always allocating a new vector here would take away the benefit of being able to reuse temporary vectors.